src/System.ServiceProcess.ServiceController/tests/ServiceBaseTests.cs

```diff
@@ -215,6 +215,18 @@ public void LogWritten_AutoLog_False()
             }
         }
 
+        [ConditionalFact(nameof(IsProcessElevated))]
+        public void PropagateExceptionFromOnStart()
+        {
+            var controller = new ServiceController(_testService.TestServiceName);
```

Comment:
did you verify this test fails without the fix above?

Reply:
(In case I need to write one of these in the future)
I'm not sure I follow what this test does. Does this test assert somehow that the exception gets propagated to the caller? (Is that the intent of this test?)

Reply:
@davidmatson I am still working on how to test the propagation of exception correctly.